Modules/Relationships/src/Relationships.Domain/DomainErrors.cs

```diff
@@ -52,4 +52,16 @@ public static DomainError CannotRequestReactivationWhenThereIsAnOpenReactivation
         return new DomainError("error.platform.validation.relationshipRequest.cannotRequestReactivationWhenThereIsAnOpenReactivationRequest",
             $"You cannot request reactivation when there is an open reactivation request.");
     }
+
+    public static DomainError RequestingIdentityDoesNotBelongToRelationship()
+    {
+        return new DomainError("error.platform.validation.relationshipRequest.requestingIdentityDoesNotBelongToRelationship",
+            $"Requesting identity does not belong to the relationship.");
+    }
+
+    public static DomainError RelationshipAlreadyDecomposed()
+    {
+        return new DomainError("error.platform.validation.relationshipRequest.relationshipAlreadyDecomposed",
+            $"Relationship has already been decomposed.");
```

Comment:
```suggestion
            $"You already decomposed this Relationship.");
```

Reply:
Changes made.

Reply:
We're really inconsistent when it comes to casing. At some point we're gonna have to fix it. For now you can leave it as you have it now.